packages/core/src/api/interactions.ts

```diff
@@ -103,7 +103,7 @@ export class InteractionsAPI {
 		body: APIInteractionResponseCallbackData & { files?: RawFile[] },
 		{ signal }: Pick<RequestData, 'signal'> = {},
 	) {
-		await this.webhooks.execute(applicationId, interactionToken, body, { signal });
+		return this.webhooks.execute(applicationId, interactionToken, { ...body, wait: true }, { signal });
```

Comment:
Should `wait` be configurable instead of hardcoded to `true`?

Reply:
why not?